chapters/hardware-software-interface/x86-architecture-family/drills/tasks/walkthrough/README.md

```diff
@@ -0,0 +1,11 @@
+# Walkthrough
+
+Access the `0-walkthrough` directory in the lab archive.
```

Comment:
Move this to guides and give it another name auch as "Discivering Assembly". And there's no lab archive. Refer to the path with the source code.